content/tools/extreme-precipitation/about.md

```diff
@@ -0,0 +1,131 @@
+<p>
+  With this tool you can explore what an extreme precipitation event
+  looks like by providing estimates of intensity and frequency of
+  extreme precipitation events. The tools and visualizations allow you
+  to examine how extreme precipitation events are likely to change in a
+  warming climate over locations of interest to you.
+</p>
+<p>
+  By default, Cal-Adapt calculates extreme values of precipitation over
+  a 2-day period, and defines an extreme event as the lowest value from
+  Annual Maximum values in the historical period (1961–1990). Users can
+  override these defaults by selecting a new “event duration” (number of
+  days over which precipitation accumulates), or by selecting a
+  different “threshold“ value that corresponds to either the 90th, 95th
+  or 99th percentiles. The tool displays the extreme events that
+  exceed the threshold in different ways. The Intensity chart shows the
+  estimated intensity of precipitation events (Return Level) for a
+  selected period (Return Period) and how it changes over the historical
+  period (1961–1990), mid-century (2035–2064) and end-century
+  (2071–2099). The other charts display the frequency of these events, the
+  timing of these events and the longest stretch of consecutive extreme
+  events.
+</p>
+<p class="h3">What is a Threshold value?</p>
```

Comment:
@mukhtyar these `<p class="h3"></p>` elements could be made to be `<h3>` elements.

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/211/commits/fa85a7530d20f241349d8e1c754ed9b316c9d85a